posting/list.go

```diff
@@ -831,8 +831,30 @@ func (l *List) Length(readTs, afterUid uint64) int {
 //     if someone is reading data between two timestamps.
 //
 //   - Drop operation can cause issues if they are rolled up. Since we are storing results at ts + 1,
-//     in dgraph.drop.op. If some operations were done then, they would be overwriten.
-//     This won't happen as the transactions should conflict out.
+//     in dgraph.drop.op. When we do drop op, we delete the relevant data first using a mutation.
+//     Then we write a record into the dgraph.drop.op. We use this record to figure out if a
+//     drop was performed. This helps us during backup, when we want to know if we need to
+//     read a given backup or not. A backup, which has a drop record, would render older backups
+//     unnecessary.
+//
+//     If we rollup the dgraph.drop.op, and store result on ts + 1, we would move the record. We
+//     want to see if there can be any issues in backup/restore due to this. Analysis is done for
```

Comment:
To ensure that there is not issue on writing at ts+1, we do the following analysis.